Makefile

```diff
@@ -215,25 +313,67 @@ oci-image-build-raw: ## Plain Jane oci build
 container: ## Build the container defined in hack/container
 	./hack/container
 
-.PHONY: check-deps
-check-deps: musl ## Check if there are any unused dependencies in Cargo.toml
-#	cargo +nightly udeps --target $(uname_m)-unknown-linux-musl --package auraed
-#	cargo +nightly udeps --package auraescript
-#	cargo +nightly udeps --package aurae-client
-#	cargo +nightly udeps --package aer
+.PHONY: spawn
```

Comment:
if nothing depends on these targets, can we drop them to keep the Makefile a bit cleaner?

Reply:
These are being used by Nóva for developing the cri services.